src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_LENGTH = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Creating an instance of `FigureSupplier` in the `Main` class is correct, but make sure that `FigureSupplier` does not contain static methods that you are using as this would be against best practices.